cargo-guppy/src/lib.rs

```diff
@@ -68,6 +70,69 @@ pub fn cmd_dups(filter_opts: &FilterOptions) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+#[derive(Debug, StructOpt)]
+pub struct ResolveCargoOptions {
+    #[structopt(long = "package", short = "p")]
+    packages: Vec<String>,
```

Comment:
IIRC I don't think this is going to do what you want. The default here is that the option `-p <packages>` will consume multiple args so this is valid `-p pkg1 pkg2 pkg3` and it will populate `packages` with those three pkgs. If instead you only want 1 package to be passed per flag then you should do this:
```suggestion
    #[structopt(long = "package", short = "p", number_of_values = 1)]
    packages: Vec<String>,
```

Reply:
Good catch!